### ISSUE TITLE
Mention of original project, Instruction to use this repo with pip

### ISSUE THREAD

**Issue by rphlo** [leriomaggio/django2-resumable#1]
It is obvious that this project is heavily inspired by https://github.com/jeanphix/django-resumable  
It would be fair to at least mention the original project.
Also you have fixed some bugs and I consider using your project instead of the original one that seems unmaintained.  
Could you update the instruction in the readme to install through pip `pip install git+https://github.com/leriomaggio/django-resumable.git`
Maybe consider renaming this project and register it in pipy to avoid confusion

**Comment by leriomaggio** [comment]
Hi @rphlo, 
   thanks a lot for the advice - gosh what a shame: I just saw this issue open. 
I wish Github could send its notification better! 

Anyway, you're totally right: if I remember correctly, I was working with `Django 2.x` and the original `django-resumable` was not supporting it - can't even remember what was the issue really. 

Plus, I think I also added Admin integration in that - which was the bit I actually needed. 

When I put it here, I was only hoping this could've been potentially useful to someone else. 
Not at all intending to replace the original `django-resumable`: I wonder why this is not a fork in the first place. 🤷 

Anyway, changed the name of the repo, and included ref. and clarified installation instructions in the README. 

Thanks for posting!
